docs/src/rules/no-useless-assignment.md

```diff
@@ -3,11 +3,25 @@ title: no-useless-assignment
 rule_type: suggestion
 related_rules:
 - no-unused-vars
+further_reading:
+- https://en.wikipedia.org/wiki/Dead_store
+- https://www.npmjs.com/package/eslint-plugin-no-useless-assign
```

Comment:
I think we can leave this off. We generally don't point to other plugins that do the same thing.